testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/health-checks/trino-health-check.sh

```diff
@@ -11,3 +11,6 @@ PID=$(jps | grep TrinoServer | cut -f1 -d ' ')
 
 # Dump process information for debugging purposes
 jstack -l $PID > /var/trino/var/log/stack.log || true
+
+# Dump Jacoco report
+java -jar /docker/jacoco-cli.jar dump --address localhost --port 57026 --destfile /var/trino/var/jacoco/jacoco-$(date +%s).exec --reset
```

Comment:
Should we make sure this is executed once the test is done? Is there a risk it would be incomplete otherwise?

Reply:
Health check is executed every 15 seconds, so that's the maximum extent of coverage loss.
I think losing up to 15 seconds of coverage isn't a big problem - configured plugins will show up in coverage during server initialization. It's just a question of what level of coverage we later consider enough to trigger a PT suite on a module change.

I tried running coverage dump on JVM shutdown first but it turned out the PT framework simply kills docker containers and doesn't execute JVM shutdown hooks, so I ended up with empty exec files.